.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 composer.lock
-vendor
\ No newline at end of file
+vendor
+.DS_STORE
```

Comment:
Ahhh, I see what you mean. 

> You should make use of global .gitignore files.

I do, but omitting said files from the project-level .gitignore means that you now depend on other devs to have appropriately configured global .gitignore files. And, realistically, there are only 3 common files of that nature: thumbs.db (Windows), .DS_STORE (Mac), .swp (*nix).

This is a very small detail, though, and I'm good either way :)